kmk/extensions/display.py

```diff
@@ -132,8 +132,9 @@ def wake(self):
 
 
 class SSD1306(DisplayBackend):
+    displayio.release_displays()
```

Comment:
Well, it might not be needed for other backends and it might ruin(might also fix?) the built in display. Seemed logical to me: as soon as you import the backend that needs it it gets executed.

Reply:
Understood. I guess we should ask upstream to fix soft reboot.